instrumentation/instaredigo/instaredigo_test.go

```diff
@@ -0,0 +1,419 @@
+// (c) Copyright IBM Corp. 2022
+
+//go:build go1.16
+// +build go1.16
+
+package instaredigo
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/gomodule/redigo/redis"
+	instana "github.com/instana/go-sensor"
+	"github.com/instana/testify/assert"
+	"github.com/instana/testify/require"
+)
+
+type MockConn struct {
```

Comment:
What do you think if we keep mock at the end of the file?

Reply:
Agreed and done